m02_programming_exercises/02_hello_world.rst

```diff
@@ -6,45 +6,11 @@ submission forms can exist anywhere and as many on one page as required.
 The automatic assessment of a submission is defined in the referenced
 YAML file.
 
-The configuration file ``docker-compose.yml`` should look something like this:  
-
-.. code-block:: yaml
-
-  version: '3'
-  
-  volumes:
-    data:
-  services:
-    grader:
-      image: apluslms/run-mooc-grader:1.4
-      volumes:
-        - data:/data
-        - /var/run/docker.sock:/var/run/docker.sock
-        - /tmp/aplus:/tmp/aplus
-        - .:/srv/courses/default:ro
-      ports:
-        - "8080:8080"
-    plus:
-      image: apluslms/run-aplus-front:1.4
-      volumes:
-        - data:/data
-      ports:
-        - "8000:8000"
-      depends_on:
-        - grader
-    acos:
-      image: apluslms/run-acos-server
-      user: $USER_ID:$USER_GID
-      ports:
-        - "3000:3000"
-      # depends_on is only used to control the start-up order of the containers so that
-      # the ACOS container would more likely be assigned the IP address hardcoded into the course configuration
-      depends_on:
-        - plus
-        - grader
-      #volumes:
-      #  - ./_data/acos/:/var/log/acos
-  
+This is the configuration file ``docker-compose.yml`` (the file is shown here using the ``include`` directive):  
```

Comment:
It's not important to state `the file is shown here using the` here. For this repo, people are anycase encouraged to read the source to understand how rst can be used for a course.